demos/apps/echo/echo-app-proxy.yaml

```diff
@@ -32,14 +32,14 @@ spec:
         pod.beta.kubernetes.io/init-containers: >
           [{
             "name": "iptables",
-            "image": "gcr.io/istio-testing/init:demo",
+            "image": "docker.io/istio/init:ceade42",
```

Comment:
Just FYI. There is a simple curl API to docker hub that allows one to query the tags for an image.